default/scripting/policies/EXOBOT_PRODUCTIVITY.focs.txt

```diff
@@ -19,9 +19,10 @@ Policy
                 TargetIndustry low = 0
                 Focus type = "FOCUS_INDUSTRY"
             ]
+            // Very bad multiplier is applied before, so it must be reverted before applying bad multiplier.
             effects = SetTargetIndustry value = Value
                         * (NamedReal name = "PLC_EXOBOT_PRODUCTIVITY_TARGET_INDUSTRY_SCALING"
```

Comment:
Note sure what the value of this NamedReal should be.
With the actual values of the multipliers it becomes 1.5, changing TargetIndustry from 0.5 * base production to 0.75 * base production

Reply:
> My only Problem is that I do not know what the NamedReal is supposed to be.
 
It defines a name for the calculation to be used in the description. Not used currently, but you could add similar to `In effect, the target industry gets multiplied by [[value PLC_EXOBOT_PRODUCTIVITY_TARGET_INDUSTRY_SCALING]]`

And maybe move the division to the end to be a little bit more clear ([[BAD_MULTIPLIER]] / [[VERY_BAD_MULTIPLIER]]). The tooltip would read something like `0.75 / 0.5`

Reply:
The problem with these multiplying effects is that you have to know in which place it happens.
E.g. Industrialism only multiplies with 1.25, but may add a lot more, because it does so after a lot of other bonuses are added.

Perhaps re-writing the effect in absolute terms makes a better fix:
`
            effects = SetTargetIndustry value = Value + Target.Population
                        * (NamedReal name = "PLC_EXOBOT_PRODUCTIVITY_TARGET_INDUSTRY_SCALING"
                                     value = ([[BAD_MULTIPLIER]] - [[VERY_BAD_MULTIPLIER]]) * [[INDUSTRY_PER_POP]])
`
That would also make the calculation less dependent on the order of effects.

Haven't notice the tooltip feature in the Pedia yet. Cool!

Reply:
The design of the good and bad industry or other resource output traits is as a multiplicative adjustment. The result of them is thus intended to be after some other effects and to also multipy those. Changing how that works is possible, but out of the scope of a fix like this.